### ISSUE TITLE
Use Tido unit testing framework

### ISSUE THREAD

**Issue by ahankinson** [music-encoding/music-encoding#306]
The Tido [mei-customization](https://github.com/tido/mei-customization) repo contains an excellent approach to building and unit testing MEI. I would like to investigate whether we can roll this into MEI proper and use it as our main build and unit test system.

The Tido approach seems to assume the unit tests are contained in the customization. I would like to see if we can incorporate the unit tests in mei-source.

@zolaemil also mentioned that the jade components of mei-customization may be phased out. We should probably get clarification on this before we start moving things over.

**Comment by pe-ro** [comment]
See also #222

**Comment by bwbohl** [comment]
see also #311

**Comment by ahankinson** [comment]
I'm going to close this as I don't think the Tido testing framework is active anymore. (That's not to say that we shouldn't have *any* testing framework, but that can be a separate issue.)
